stl/inc/format

```diff
@@ -1758,7 +1767,19 @@ private:
     void _Flush() {
         auto _Size = this->_Size();
         this->_Clear();
-        _Output = _STD _Copy_unchecked(_Data, _Data + this->_Limit(_Size), _STD move(_Output));
+        const auto _End = _Data + this->_Limit(_Size);
+
+        if constexpr (_Is_specialization_v<_OutputIt, back_insert_iterator>) {
+            using _Container = typename _OutputIt::container_type;
+            if constexpr (_Is_specialization_v<_Container, basic_string> || _Is_specialization_v<_Container, vector>) {
+                auto& _Cont = *_Back_insert_iterator_container_access<_Container>{_Output}.container;
+                _Cont.insert(_Cont.end(), _Data, _End);

```

Comment:
Should we move this optimization to `_STD _Copy_unchecked` and `_RANGES _Copy_unchecked`?

Reply:
I think we should leave `_Copy_unchecked` to be "dumb", but will wait for a call from a maintainer.

Reply:
Leaning towards "dumb", but I could be convinced otherwise. But not in this PR.